src/lick/lick.py

```diff
@@ -16,6 +16,7 @@ def interpol(
     field: np.ndarray,
     *,
     method: str = "nearest",
+    method_background: str = "nearest",
```

Comment:
It'd be nice to make it easy to use the same method in all interpolations by default (while still allowing that a different method be specified for the background if needed). Here's a proposed implementation:
```suggestion
    method_background: Optional[str] = None,
```
And in the function's body
```python
    if method_background is None:
        method_background = method
```

Reply:
note that my suggestion is *not* backward compatible, but yours is !

Reply:
Good point, but after thinking about it, it is probably better to have the most conservative (compared to the original data) interpolation by default ("nearest"), and if you want to do fancy stuff ("linear", "cubic") you have to do it explicitly. Do you know an efficient way to do that? Thanks for your review!

Reply:
I didn't realise I was suggesting a backward-incompatible change as a wrote it, and now I hate it. Your patch fine as is !